sources/packages/forms/Dockerfile

```diff
@@ -32,13 +30,12 @@ ENV NPM_PACKAGES=/.npm-packages \
 RUN echo "prefix = $NPM_PACKAGES" >> ~/.npmrc
 
 # Include details of the required dependencies
-COPY ./package.json $NPM_PACKAGES/
-COPY ./package-lock.json $NPM_PACKAGES/
-
-# Use "Continuous Integration" to install as-is from package-lock.json
-RUN npm ci --prefix=$NPM_PACKAGES
+RUN mkdir -p $NPM_PACKAGES/
+# Include details of the required dependencies
+RUN cp package.json $NPM_PACKAGES/
```

Comment:
why package-lock is not copied?

Reply:
when we are installing via the package.json using npm i the package-lock.json is implicitly unused. So removed it and it may cause libraries, version mismatch conflict.